MantidQt/SliceViewer/src/SliceViewer.cpp

```diff
@@ -670,6 +670,13 @@ void SliceViewer::updateDimensionSliceWidgets() {
     if (w > maxUnitsWidth)
       maxUnitsWidth = w;
 
+    // if the workspace is already binned, update the interface with the
+    // existing number of bins
+    if (boost::dynamic_pointer_cast<IMDHistoWorkspace>(m_ws)) {
```

Comment:
@martyngigg @OwenArnold I couldn't see an existing method for it either. I'd be happy to create an issue for it?